contracts/package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "truffle-test": "./scripts/truffle-test.sh",
     "truffle-test-v": "./scripts/truffle-test.sh --verbose-rpc",
-    "ganache": "docker run --name audius_ganache_cli -d -p 8545:8545 trufflesuite/ganache-cli:latest -h 0.0.0.0 -l 8000000",
+    "ganache": "docker run --name audius_ganache_cli -d -p 8545:8545 trufflesuite/ganache-cli:latest -h 0.0.0.0 -l 8000000 --acctKeys contracts-ganache-accounts.json",
```

Comment:
is this json file generated when this command is run?

Reply:
this is interesting, it is generated when the ganache command is run BUT that happens inside the container, which is why the utils functions below call docker exec <containerName> cat <fileName> and get the secret values from there

Reply:
ahhhh yes that makes sense